apps/web/src/components/editor/index.tsx

```diff
@@ -703,8 +703,10 @@ function EditorChrome(props: PropsWithChildren<EditorProps>) {
       );
 
       requestAnimationFrame(() => {
-        editor.style.marginLeft = `-${negativeSpace}px`;
-        editor.style.marginRight = `-${negativeSpace}px`;
+        if (!isMobile() && !isTablet()) {
+          editor.style.marginLeft = `-${negativeSpace}px`;
+          editor.style.marginRight = `-${negativeSpace}px`;
+        }
```

Comment:
have to remove `marginLeft` for small screens otherwise the collapse heading icon gets cut off
removing `marginRight` for consistency